locale/nl/docs/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Documentatie
+layout: docs.hbs
+labels:
+lts: LTS
+---
+
+# Documentatie
+
+Er zijn verschillende soorten documentatie beschikbaar op onze website:
+
+* API-referentiedocumentatie
+* ES6-features
+* Handleiding
+
+## API-referentiedocumentatie
+
+De [API-referentiedocumentatie](https://nodejs.org/api/) biedt gedetailleerde informatie over functies en objecten in Node.js. Het geeft aan welke argumenten geaccepteerd kunnen worden door een methode, de return-waarde van een methode en informatie over errors die gekoppeld zijn met deze methode. Het geeft ook aan welke methoden beschikbaar zijn binnen de verschillende versies van Node.js.
+
+Deze documentatie behandelt enkel de ingebouwde modules van Node.js. Het behandelt geen community-modules.
+
+<div class="highlight-box">
+
+### Op zoek naar API-documentatie van vorige versies?
+
+{#var nodeVersionLinks}* [Alle versies](https://nodejs.org/docs/)
+
+</div>
+
+## ES6-features
+
+De [ES6-sectie](/en/docs/es6/) beschrijft de drie ES6-feature groepen en de standaard features binnen Node.js. Het toont u ook welke versie van V8 bij een bepaalde Node.js-versie hoort.
```

Comment:
nit: I don't think the `u` is needed in the last sentence

Reply:
Additionally, "het" cannot refer to the antecedent "de ES6-sectie". 'Sectie' is feminine so it should be 'ze' or 'zij'.